graphql_compiler/tests/test_emit_output.py

```diff
@@ -289,3 +294,239 @@ def test_datetime_output_representation(self):
 
         received_match = emit_gremlin.emit_code_from_ir(ir_blocks, None)
         compare_gremlin(self, expected_gremlin, received_match)
+
+
+class EmitCypherTests(unittest.TestCase):
+    def setUp(self):
+        """Disable max diff limits for all tests."""
+        self.maxDiff = None
+
+    def test_simple_immediate_output(self):
+        # corresponds to:
+        # graphql_string = '''{
+        #     Foo {
```

Comment:
As much as possible, I'd try to keep all tests (even these ones) in sync with the test schema -- that makes it more likely that the various test helpers will be able to be reused. Not doing that to start with this module was a mistake, and it's a bit late to rectify it now.

Reply:
done for these! Other test changes will come soon.